<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeasonalDecompositionDock</class>
 <widget class="QWidget" name="SeasonalDecompositionDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>981</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lName">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="TimedLineEdit" name="leName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lComment">
     <property name="text">
      <string>Comment:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="ResizableTextEdit" name="teComment">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lData">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lXColumn">
     <property name="text">
      <string>X-Data:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lYColumn">
     <property name="text">
      <string>Y-Data:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lOptions">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lMethod">
     <property name="text">
      <string>Method:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="KComboBox" name="cbMethod">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lSTLPeriod">
     <property name="text">
      <string>Period:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QSpinBox" name="sbSTLPeriod"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lSTLRobust">
     <property name="text">
      <string>Robust:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QCheckBox" name="chbSTLRobust">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="lSTLSeasonalLength">
     <property name="text">
      <string>Seasonal Length:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QSpinBox" name="sbSTLSeasonalLength">
     <property name="minimum">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="14" column="2" rowspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSpinBox" name="sbSTLTrendLength">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSTLTrendLengthAuto">
       <property name="text">
        <string>Auto</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="16" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSpinBox" name="sbSTLLowPassLength">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimum">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSTLLowPassLengthAuto">
       <property name="text">
        <string>Auto</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="18" column="2">
    <widget class="QComboBox" name="cbSTLSeasonalDegree"/>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="lSTLTrendDegree">
     <property name="text">
      <string>Trend Degree:</string>
     </property>
    </widget>
   </item>
   <item row="19" column="2">
    <widget class="QComboBox" name="cbSTLTrendDegree"/>
   </item>
   <item row="20" column="2">
    <widget class="QComboBox" name="cbSTLLowPassDegree"/>
   </item>
   <item row="22" column="0">
    <widget class="QLabel" name="lSTLSeasonalJump">
     <property name="text">
      <string>Seasonal Jump:</string>
     </property>
    </widget>
   </item>
   <item row="22" column="2">
    <widget class="QSpinBox" name="sbSTLSeasonalJump"/>
   </item>
   <item row="23" column="0">
    <widget class="QLabel" name="lSTLTrendJump">
     <property name="text">
      <string>Trend Jump:</string>
     </property>
    </widget>
   </item>
   <item row="23" column="2">
    <widget class="QSpinBox" name="sbSTLTrendJump"/>
   </item>
   <item row="24" column="0">
    <widget class="QLabel" name="lSTLLowPassJump">
     <property name="text">
      <string>Low-Pass Jump:</string>
     </property>
    </widget>
   </item>
   <item row="24" column="2">
    <widget class="QSpinBox" name="sbSTLLowPassJump"/>
   </item>
   <item row="2" column="0" colspan="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="2">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="17" column="0">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>38</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="lSTLSeasonalDegree">
     <property name="text">
      <string>Seasonal Degree:</string>
     </property>
    </widget>
   </item>
   <item row="20" column="0">
    <widget class="QLabel" name="lSTLLowPassDegree">
     <property name="text">
      <string>Low-Pass Degree:</string>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="25" column="0">
    <spacer name="verticalSpacerGeneral">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>24</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="lSTLLowPassLength">
     <property name="text">
      <string>Low-Pass Length:</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" rowspan="2">
    <widget class="QLabel" name="lSTLTrendLength">
     <property name="text">
      <string>Trend Length:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>frontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>TimedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>frontend/widgets/TimedLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>teComment</tabstop>
  <tabstop>cbMethod</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
